gltfTutorial/gltfTutorial_004_ScenesNodesCamerasAnimations.md

```diff
@@ -0,0 +1,273 @@
+# Scenes, nodes, cameras and animations
+
+## Scenes
+
+The [`scene`](https://github.com/KhronosGroup/glTF/tree/master/specification#reference-scene) is the entry point for the description of the scene that is stored in the glTF. When parsing a glTF JSON file, the traversal of the scene structure will start here.
+
+There may be multiple scenes stored in one glTF file, but in many cases, there will only be a single scene, which then also is the default scene. Each scene contains an array of `nodes`, which are the IDs of the root nodes of the scene graphs. Again, there may be multiple root nodes, forming different hierarchies, but in many cases, the scene will have a single root node.
+
+So the most simple scene description may look as follows:
+
+```javascript
+"scene" : "defaultSceneId",
+"scenes" : {
+    "defaultSceneId": {
+        "nodes": [
+            "rootNodeId",
+        ],
+    }
+},
+"nodes" : {
+    "rootNodeId": { ... }
+}
+```
+
+
+## Nodes forming the scene graph
+
+Each [`node`](https://github.com/KhronosGroup/glTF/tree/master/specification#reference-node) is one element of a hierarchy of nodes, and together they define the structure of the scene as a scene graph.  
+
+![Scene graph](images/sceneGraph.png)
+
+Each of the nodes that are given in the `scene` can be traversed, recursively visiting all its children, to process all elements that are attached to the nodes. The simplified pseudocode for this traversal may look as follows:
+
+```
+traverse(node) {
+    processCamera(node.camera);
+    processMeshes(node.meshes);
+    processOtherElements(node); // Skin, joint, skeleton - discussed later
+    for each (child in node.children) {
+        traverse(child);
+    }
+}
+```
+
+In practice there will some additional information be required for the traversal: The processing of some elements that are attached to nodes will require information about *which* node they are attached to. Additionally, the information about the transforms of the nodes have to be accumulated during the traversal. This will be detailed in the following section.
+
+
+### Local and global transforms
+
+Each node can have a transform. Such a transform will usually define a translation, rotation or a scaling. This transform will be applied to all elements that are attached to the node itself and to all its child nodes. The hierarchy of nodes thus allows to structure the translations, rotations and scalings that are applied to the scene elements.

```

Comment:
"usually define a translation, rotation or a scaling" -> "define a translation, rotation and/or scale"